<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Haro</class>
 <widget class="QMainWindow" name="Haro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>413</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Haro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="bodyImage">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>4</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../res.qrc">:/body/res/images/appearance/body/def_body.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="eyeImage">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>4</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../res.qrc">:/eye/res/images/appearance/eyes/def_eyes.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="stripeImage">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>4</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../res.qrc">:/stripe/res/images/appearance/stripe.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="earImage">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>4</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../res.qrc">:/ear/res/images/appearance/ears/def_ears1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>104</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border:4px solid black;background-color:rgb(200,210,255);border-radius: 10px;}QPushButton::hover{background-color:rgb(170,200,255);}QPushButton:pressed{background-color:rgb(60,70,200);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../res.qrc">
      <normaloff>:/icon/res/images/icon/close.png</normaloff>:/icon/res/images/icon/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="dressButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border:4px solid black;background-color:rgb(200,210,255);border-radius: 10px;}QPushButton::hover{background-color:rgb(170,200,255);}QPushButton:pressed{background-color:rgb(60,70,200);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../res.qrc">
      <normaloff>:/icon/res/images/icon/dress.png</normaloff>:/icon/res/images/icon/dress.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="moreButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>237</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border:4px solid black;background-color:rgb(200,210,255);border-radius: 10px;}QPushButton::hover{background-color:rgb(170,200,255);}QPushButton:pressed{background-color:rgb(60,70,200);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../res.qrc">
      <normaloff>:/icon/res/images/icon/more.png</normaloff>:/icon/res/images/icon/more.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="minButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>304</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border:4px solid black;background-color:rgb(200,210,255);border-radius: 10px;}QPushButton::hover{background-color:rgb(170,200,255);}QPushButton:pressed{background-color:rgb(60,70,200);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../res.qrc">
      <normaloff>:/icon/res/images/icon/min.png</normaloff>:/icon/res/images/icon/min.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="settingButton">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>104</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border:4px solid black;background-color:rgb(200,210,255);border-radius: 10px;}QPushButton::hover{background-color:rgb(170,200,255);}QPushButton:pressed{background-color:rgb(60,70,200);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../res.qrc">
      <normaloff>:/icon/res/images/icon/setting.png</normaloff>:/icon/res/images/icon/setting.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="musicButton">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>170</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border:4px solid black;background-color:rgb(200,210,255);border-radius: 10px;}QPushButton::hover{background-color:rgb(170,200,255);}QPushButton:pressed{background-color:rgb(60,70,200);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../res.qrc">
      <normaloff>:/icon/res/images/icon/music.png</normaloff>:/icon/res/images/icon/music.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="gameButton">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>237</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border:4px solid black;background-color:rgb(200,210,255);border-radius: 10px;}QPushButton::hover{background-color:rgb(170,200,255);}QPushButton:pressed{background-color:rgb(60,70,200);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../res.qrc">
      <normaloff>:/icon/res/images/icon/game.png</normaloff>:/icon/res/images/icon/game.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="calendarButton">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>304</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border:4px solid black;background-color:rgb(200,210,255);border-radius: 10px;}QPushButton::hover{background-color:rgb(170,200,255);}QPushButton:pressed{background-color:rgb(60,70,200);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../res.qrc">
      <normaloff>:/icon/res/images/icon/calendar.png</normaloff>:/icon/res/images/icon/calendar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <zorder>bodyImage</zorder>
   <zorder>eyeImage</zorder>
   <zorder>earImage</zorder>
   <zorder>stripeImage</zorder>
   <zorder>closeButton</zorder>
   <zorder>dressButton</zorder>
   <zorder>moreButton</zorder>
   <zorder>minButton</zorder>
   <zorder>settingButton</zorder>
   <zorder>musicButton</zorder>
   <zorder>gameButton</zorder>
   <zorder>calendarButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
